linera-service/src/server.rs

```diff
@@ -288,7 +291,7 @@ fn make_server_config<R: CryptoRng>(
     options: ValidatorOptions,
 ) -> anyhow::Result<persistent::File<ValidatorServerConfig>> {
     let validator_keypair = ValidatorKeypair::generate_from(rng);
-    let account_secret = AccountSecretKey::generate_from(rng);
+    let account_secret = AccountSecretKey::from(Ed25519SecretKey::generate_from(rng));
```

Comment:
Would it make sense to have an `AccountSecretKey::generate_from` for tests, that chooses a random key type?

Reply:
I had that initially but then thought it might not be desirable to _not_ know what type of keys you'll get. Although I guess that's OK if you can choose a specific key scheme anyway (like this).

I'm not sure if it's worth adding now, given that the code has been updated and compiles.